### ISSUE TITLE
Card has poor compatibility with assistive technology

### ISSUE THREAD

**Issue by adampage** [adampage/github-cli-test#11]
The HTML+CSS implementation of our Breadcrumb component has poor compatibility with assistive technology and is lacking in semantic quality. To improve it, we can make the following changes:

- Wrap the card in an `<article>`.
- Include an appropriately-leveled `<h#>` heading element for the Card title.
- The heading must precede all Card content. Its visual order can be manipulated with CSS.
- Allow adopting context to specify image as 1) decorative, or 2) informative with alternative text.
- If the card includes a checkbox, then it must be described by a `<label>` element; likely the Card's title.

Heydon Pickering's [inclusive Card component](https://inclusive-components.design/cards/) is a good source for imitation.

## Applicable WCAG Success Criteria

- [`1.1.1` Non-text Content — Level A](https://www.w3.org/WAI/WCAG21/quickref/#non-text-content)
- [`1.3.1` Info and Relationships — Level A](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)
- [`1.3.2` Meaningful Sequence — Level A](https://www.w3.org/WAI/WCAG21/quickref/#meaningful-sequence)
- [`4.1.2` Name, Role, Value — Level A](https://www.w3.org/WAI/WCAG21/quickref/#name-role-value)
